packages/@aws-cdk/aws-events-targets/lib/api-gateway.ts

```diff
@@ -0,0 +1,141 @@
+import * as api from '@aws-cdk/aws-apigateway';
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sqs from '@aws-cdk/aws-sqs';
+import { addToDeadLetterQueueResourcePolicy, singletonEventRole } from './util';
+
+/**
+ * Customize the API Gateway Event Target
+ */
+export interface ApiGatewayProps {
+
+  /**
+   * The method for api resource invoked by the rule.
+   *
+   * @default '*' that treated as ANY
+   */
+  readonly method?: string;
+
+  /**
+   * The api resource invoked by the rule.
+   * We can use wildcards('*') to specify the path. In that case,
+   * an equal number of real values must be specified for pathParameterValues.
+   *
+   * @default '/'
+   */
+  readonly path?: string;
+
+  /**
+   * The deploy stage of api gateway invoked by the rule.
+   *
+   * @default the value of deploymentStage.stageName of target api gateway.
+   */
+  readonly stage?: string;
+
+  /**
+   * The headers to be set when requesting API
+   *
+   * @default no header parameters
+   */
+  readonly headerParameters?: { [key: string]: (string) };
+
+  /**
+   * The path parameter values to be used to
+   * populate to wildcards("*") of requesting api path
+   *
+   * @default no path parameters
+   */
+  readonly pathParameterValues?: string[];
+
+  /**
+   * The query parameters to be set when requesting API.
+   *
+   * @default no querystring parameters
+   */
+  readonly queryStringParameters?: { [key: string]: (string) };
+
+  /**
+   * Message Group ID for messages sent to this queue.
+   *
+   * Required for FIFO queues, leave empty for regular queues.
+   *
+   * @default - no message group ID (regular queue)
+   */
+  readonly messageGroupId?: string;
+
+  /**
+   * The message to send to the queue.
+   *
+   * Must be a valid JSON text passed to the target queue.
+   *
+   * @default the entire EventBridge event
+   */
+  readonly message?: events.RuleTargetInput;
```

Comment:
Does this not need to be removed? 

Where DOES the payload go?

Reply:
Should this be called something like `postBody` ?

Reply:
You are right. This will send to API as post body. I fixed.